feature/account/src/commonMain/kotlin/org/mifos/mobile/feature/account/account/screens/AccountsScreen.kt

```diff
@@ -10,44 +10,44 @@
 package org.mifos.mobile.feature.account.account.screens
 
 import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.pulltorefresh.PullToRefreshBox
+import androidx.compose.material3.pulltorefresh.rememberPullToRefreshState
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.LaunchedEffect
 import androidx.compose.runtime.getValue
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.platform.LocalContext
-import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.tooling.preview.PreviewParameter
-import androidx.compose.ui.tooling.preview.PreviewParameterProvider
-import androidx.hilt.navigation.compose.hiltViewModel
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
-import com.mifos.library.pullrefresh.PullRefreshIndicator
-import com.mifos.library.pullrefresh.pullRefresh
-import com.mifos.library.pullrefresh.rememberPullRefreshState
-import org.mifos.mobile.core.common.Network
+import mifos_mobile.feature.account.generated.resources.Res
+import mifos_mobile.feature.account.generated.resources.feature_account_empty_loan_accounts
+import mifos_mobile.feature.account.generated.resources.feature_account_empty_savings_accounts
+import mifos_mobile.feature.account.generated.resources.feature_account_empty_share_accounts
+import mifos_mobile.feature.account.generated.resources.feature_account_error_black
+import org.jetbrains.compose.resources.vectorResource
+import org.jetbrains.compose.ui.tooling.preview.Preview
+import org.jetbrains.compose.ui.tooling.preview.PreviewParameter
+import org.jetbrains.compose.ui.tooling.preview.PreviewParameterProvider
+import org.koin.compose.viewmodel.koinViewModel
 import org.mifos.mobile.core.designsystem.theme.MifosMobileTheme
 import org.mifos.mobile.core.model.entity.accounts.loan.LoanAccount
 import org.mifos.mobile.core.model.entity.accounts.savings.SavingAccount
 import org.mifos.mobile.core.model.entity.accounts.share.ShareAccount
 import org.mifos.mobile.core.ui.component.EmptyDataView
 import org.mifos.mobile.core.ui.component.MifosErrorComponent
 import org.mifos.mobile.core.ui.component.MifosProgressIndicatorOverlay
-import org.mifos.mobile.feature.account.R
 import org.mifos.mobile.feature.account.utils.AccountState
 import org.mifos.mobile.feature.account.viewmodel.AccountsViewModel
 
 @Composable
 internal fun AccountsScreen(
+    isNetworkConnected: Boolean,
```

Comment:
Inject NetworkMonitor directory in AccountsViewModel and observe as AccountsViewModel in here and composable and use